dev-python/pyilmbase/pyilmbase-2.5.2.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+CMAKE_ECLASS=cmake
+inherit cmake-multilib python-r1
+
+DESCRIPTION="IlmBase Python bindings"
+HOMEPAGE="https://www.openexr.com"
+SRC_URI="https://github.com/AcademySoftwareFoundation/openexr/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
```

Comment:
`LICENSE` is in the same block as `SLOT`: https://devmanual.gentoo.org/eclass-reference/ebuild/#lbAQ